api/repositories/service_instance_repository_test.go

```diff
@@ -889,8 +889,8 @@ var _ = Describe("ServiceInstanceRepository", func() {
 			serviceInstance = createServiceInstanceCR(ctx, k8sClient, prefixedGUID("service-instance"), space.Name, "the-service-instance", prefixedGUID("secret"))
 
 			deleteMessage = repositories.DeleteServiceInstanceMessage{
-				GUID:      serviceInstance.Name,
-				SpaceGUID: space.Name,
+				GUID:  serviceInstance.Name,
+				Purge: false,
```

Comment:
`false` is the default value for booleans, so no need to set it explicitly. As per testing with `purge=true`, the test could simply verify that the finalizer on the `CFServiceInstance` object is removed (as this is what the repository is doing)